README.md

```diff
@@ -146,6 +146,12 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
   - Custom load balancing algorithms: You can use custom load balancing algorithms during the `balancer` phase.
   - Custom routing: Support users to implement routing algorithms themselves.
 
+- **Multi-Language support**
+  - Apache APISIX is a multi-language gateway for plugin development and provides support via `WASM` and `RPC`.
+  ![Multi Language Support into Apache APISIX](docs/assets/images/apisix-multi-lang-support.png)
+  - The WASM or WebAssembly, is the modern way. If we look at the left part, APISIX can load and run WASM bytecode via APISIX wasm plugin, so developers only need to write the code according to the SDK and then compile it into a WASM bytecode file and put it in the APISIX folder. As of now, APISIX has support for [Golang SDK](https://github.com/golang/go/wiki/WebAssembly#getting-started) and [Rust SDK](https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm) - more language SDKs will be supported in the future.
```

Comment:
The SDK should be https://github.com/proxy-wasm/spec#sdks.
And there are not just Go/Rust.
And there is no requirement to `put it in the APISIX folder`.
See https://github.com/apache/apisix/blob/master/docs/en/latest/wasm.md

Reply:
ping @bisakhmondal

Reply:
Ah, I got it now. The sdk means - the proxy wasm SDK. Updated accordingly. Is it okay now?
cc @spacewander